stockflux-securities-master/src/components/reusableComponents/ToggleSwitch.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./ToggleSwitch.css";
+
+const TextField = ({ value, id, checked, onChange }) => {
+  return (
+    <div>
```

Comment:
If the label is removed then you will have a single component (`<input />`) to return so you won't need a `div`. As a tip, if you want to return elements in your components, it's possible to wrap them with `<React.Fragment>...</React.Fragment>`, or better use its shorthand syntax `<>...</>`

Reply:
Okay I've changed the <div>s to be <> fragment tags.